StripeCore/StripeCore/Source/Localization/STPLocalizationUtils.swift

```diff
@@ -37,7 +37,7 @@ import Foundation
         return false
     }
 
-    static let UnknownString = "STPStringNotFound"
+    static let UnknownString = "STPSTRINGNOTFOUND"
```

Comment:
when checking `userTranslation != UnknownString`, should we:
```
userTranslation.uppercased() != UnknownString
```

Reply:
Not anymore, as we're just passing the uppercase string in. So it should always be uppercase.

Reply:
My first pass was to check `.uppercased()`, but then I realized we could just change the string

Reply:
Assuming this is good for all versions of iOS, lgtm!